README.md

```diff
@@ -1 +1,100 @@
-# My_capstone_project_001
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)<br>
+- [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+-[LIVE DEMO](#live-demo)<br>
+-[LOOM VIDEO](#loom-video)<br>
+💻 Getting Started<br>
+-Setup<br>
+-Prerequisites<br>
+- [👥 Authors](#authors)<br>
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Capstone project ] <a name="about-project"></a>
+
+
+*[Capstone project]* This project is a final project that I worked on to demonstrate the skills and knowledge I have acquired during the first module of the program.
+it's about : African women in tech group <br> launches to support diversity<br>
+**Key Features**<br>
+simple Capstone project built with Bootstrap,html5,CSS and simple Javascript With linter configurattion.<br>
+
+[Set up linters]<br>
+[Add navigation bar for mobile and Desktop version]<br>
+[Add intro header section for mobile and Desktop version]<br>
+[Add media queries(Responsiveness)]<br>
+**Tech Stack**<br>
+This portfolio project, was built with html5 and css3
+
+Client<br>
+**.Bootstrap**
+**.Javascript**
+**.HTML**
+**.CSS**
+## 🚀 LIVE DEMO  <a name="live-demo"></a>
+To get deployed version of my Capstone project<br>
+visit <a href="https://ismailmunyentwari9.github.io/My_capstone_project_001/">Capstone project:001</a>
+
+## 🚨 LOOM VIDEO  <a name="loom-video"></a>
+If you want to watch full Presantation of my project through video<br>
+visit <a href="https://www.loom.com/share/66fab321bf4b4e478211a033741c099b">**LOOM**</a>
```

Comment:
- [ Required ] Great job in recording a video presentation of your project, however you forgot :smiley: to include the following video requirements.
  - A demo of the project, showing the pages in different screen sizes.
  - Highlight about a piece of the code the student finds interesting or is proud of making.

## Project requirements
![Screenshot from 2023-01-12 17-36-29](https://user-images.githubusercontent.com/16392046/212095241-4dc9409e-d276-4e49-ba71-2a496ab3400b.png)